qiskit_machine_learning/algorithms/classifiers/vqc.py

```diff
@@ -47,6 +48,8 @@ class VQC(NeuralNetworkClassifier):
 
     def __init__(
         self,
+        *,
+        sampler: BaseSampler = None,
```

Comment:
I don't think we can easily convert VQC to keyword only parameters. It should be a transition period when positional are accepted and generate warnings, while keywords are accepted without warnings.

Reply:
As such `sampler` needs to go last in the list here, as we needed to do when we have altered other algorithms in place elsewhere